docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -10,7 +10,7 @@ with a variety of applications, the users must be able to grant
 privileges for security-sensitive operations.
```

Comment:
In line 7-8, "Since Tizen, as an open mobile platform" 

I think "mobile" should be removed.

Reply:
done